flexmeasures/api/v3_0/tests/test_assets_api.py

```diff
@@ -118,6 +119,28 @@ def test_get_assets(
         assert turbine["account_id"] == setup_accounts["Supplier"].id
 
 
+@pytest.mark.parametrize("requesting_user", ["test_admin_user@seita.nl"], indirect=True)
+def test_fetch_asset_sensors(client, setup_api_test_data, requesting_user):
```

Comment:
Please test two options: without pagination (done here) and with.
You can use mostly the same code and differentiate with `@pytest.mark.parametrize` (e.g. the test above parameterised both who the user is and the status code) you can parameterise a flag `use_pagination` and when it is True, send a `page` parameter and check if the response has the right structure.

Reply:
But remember we said that since this is a new API we would make the pagination implemented by default since there is nothing to try to be backward compatible with. This means that even if `page` is not passed the API will default to setting `page=1` and `per_page=`10` and the response will be the same (that is including `["data"]`

Reply:
Okay, easier to test!